common/story_cycles/wc_story_cycle_thunder_king.txt

```diff
@@ -0,0 +1,159 @@
+	# common/on_action/game_start.txt
+	# common/on_action/yearly_on_actions.txt
+	# common/game_rules/wc_game_rules.txt
+	# common/scripted_character_templates/wc_mogu_templates.txt
+	# common/scripted_effects/wc_pandaria_effects.txt
+	# events/decisions_events/wc_major_pandaria_events.txt
+	
+	### Can be initiated through:
+	# - yearly_on_action
+	# - revive_lei_shen_decision
+	# - zandalari_lei_shen_decision
+	
+	on_setup = {
+		if = { # Make sure lightning don't strike twice
+			limit = {
+				NOT = {
+					is_target_in_global_variable_list = {
+						name = unavailable_unique_decisions
+						target = flag:revive_lei_shen_decision
+					}
+				}
+			}
+			add_to_global_variable_list = {
+				name = unavailable_unique_decisions
+				target = flag:revive_lei_shen_decision
+			}
+		}
+		story_owner = {
+			save_scope_as = lei_shen
+			var:reviver ?= {
+				save_scope_as = reviver
+			}
+		}
+		
+		# Handles Court setup, troops, etc
+		story_owner = {
+			trigger_event = pandaria_unification.0101
+		}
+		
+		## Handles Mogu-Zandalari Pact
+		#if = {
+		#	limit = {
+		#		scope:reviver.culture = { has_cultural_pillar = heritage_zulite }
+		#	}
+		#	story_owner = {
+		#		trigger_event = {
+		#			id = pandaria_unification.0302
+		#			days = 10
+		#		}
+		#	}
+		#	scope:reviver = {
+		#		trigger_event = {
+		#			id = pandaria_unification.0302
+		#			days = 10
+		#		}
+		#	}
+		#}
+		
+	}
+	
+	# Disband his realm, sends news event
+	on_end = {
+		story_owner = { lei_shen_death_effect = yes }
+	}
+
+	on_owner_death = {
+		end_story = yes
+	}
+	
+	# AI gets Troops + Gold periodically if they are weak
+	effect_group = {
+		years = { 5 8 }
+		
+		trigger = {
+			story_owner = {
+				is_ai = yes
+				is_independent_ruler = yes
+				#highest_held_title_tier = tier_empire
+			}
+		}
+		
+		triggered_effect = {
+			trigger = {
+				always = yes
+			}
+			effect = {
+				story_owner = {
+					trigger_event =  pandaria_unification.0104
+				}
+			}
+		}
+	}
+	
+	# Fire event for AI Thunder King to invade another ruler on Pandaria
+	effect_group = {
+		years = { 1 3 }
+
+		trigger = {
+			story_owner = {
+				is_ai = yes
+				is_independent_ruler = yes
+				is_able_to_wage_war_trigger = yes
+				NOT = { # Pandaria not united
+					is_target_in_global_variable_list = {
+						name = unavailable_unique_decisions
+						target = flag:create_pandaria_decision
+					}
+				}
+			}
+		}
+
+		triggered_effect = {
+			trigger = {
+				always = yes
+			}
+			effect = {
+				story_owner = {
+					trigger_event =  pandaria_unification.0105
+				}
+			}
+		}
+	}
+	
+	# Story handling in case of succession weirdness
+	effect_group = {
+		days = 30
+
+		trigger = {
+			story_owner = {
+				OR = {
+					is_landed = no
+					is_independent_ruler = no
+				}
+			}
+		}
+
+		triggered_effect = {
+			trigger = {
+				always = yes
+			}
+			effect = {
+				story_owner = {
+					death = { death_reason = death_battle }
+				}
+				end_story = yes
+			}
+		}
+	}
```

Comment:
what i'm getting at is, the crisis is tied to this story cycle but is only considered a crisis if Lei Shen is an emperor. so the crisis is tied to this story cycle, but this story cycle is not tied to the crisis? should the story cycle not end if Lei Shen isn't an emperor anymore? he will keep using his crazy CB to conquer territory until he's an emperor again and then the crisis resumes?

Reply:
my opinion is that one of these effect groups, or a new effect group, should check if Lei Shen no longer has any empire title and then end this story cycle turning him into a normal run of the mill ruler.

Reply:
The story cycle continues as long as Lei Shen is alive. However, if he looses his empire title, the crisis is "on pause" until he gets one back. If he is vassalized or unlanded, he is immediately killed via effect and the crisis ends properly same as if he died from something else. The story cycle should only end when he dies, since some stuff related to his death fires if he is defeated.

I dont think he should ever become a "normal ruler" permanently, the crisis should end with his death. (I was also really confused why that isnt the case with the lich king, who sometimes just hangs out as a duke), I'd rather add a line that also kills him if he looses his empire title - though this would be very punishing for players that get hit by a dissolution faction etc, which is why add it previously.